blocks/targeted-block/targeted-block.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/no-unresolved */
+import { TargetedBlock } from '@dropins/storefront-personalization/containers/TargetedBlock.js';
+import { render } from '@dropins/storefront-personalization/render.js';
+import { readBlockConfig } from '../../scripts/aem.js';
+import { loadFragment } from '../fragment/fragment.js';
+
+function prepareIds(providedIds) {
+  return providedIds.split(',').map((num) => btoa(num.trim()));
```

Comment:
Shouldn't we `trim()` also `providedIds`? What happens if the included values are separated with `, ` (with a space after each comma)?

Reply:
it is not necessary as the result trim will handle that. f the included values are separated with `, ` the space will be trimmed.